libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -158,6 +158,7 @@ void AP_Camera_Backend::send_camera_information(mavlink_channel_t chan) const
     const char cam_definition_uri[140] {};
     const uint32_t cap_flags = CAMERA_CAP_FLAGS_CAPTURE_IMAGE;
     const float NaN = nanf("0x4152");
+    const uint8_t gimbal_device_id = 0;
```

Comment:
we can use _instance for now.  We will add a parameter for this in the near future though, it has almost reached the top of the To-Do list.

Reply:
But _instance in that file will refer to Camera instance, which is unrelated with gimbal_device_id right? Isn't it better to leave it at 0, and deal with it with a parameter later on if needed? Thanks!

Reply:
Yes, you're right.  maybe just lose the local const and just set it to zero directly a little further down?

Reply:
Sure! I wasn't sure if you guys wanted to implement the parameter now so I lazily left it prepared to it. Should be fixed now!